platform/kube/kube_test.go

```diff
@@ -245,6 +240,48 @@ func TestServices(t *testing.T) {
 	}
 }
 
+func TestProxyConfig(t *testing.T) {
+	cl := makeClient(t)
+	ns := makeNamespace(cl.client, t)
+	defer deleteNamespace(cl.client, ns)
+
+	rule := &proxyconfig.RouteRule{
+		RouteRule: &proxyconfig.RouteRule_Http{
+			Http: &proxyconfig.HttpRouteRule{
```

Comment:
Please add dst_service here. We may need to do immediate validation in tpr code to see if the user provided service is valid or invalid